### PR TITLE
RA-664

### PR DESCRIPTION
**Pull request by erdody**

### CODE REVIEW

**Comment by mauri** [comment]
LGTM
Could you add a small paragraph on top of the README file saying how to use this feature and also how to build it? (now that we have a new vendor.sh)

I'll put a proper README.md with our changes of network/storage (including yours) and place it in the 1.9.1-medallia branch. Then I'll change the github default branch to that. So it'll more friendly with visitors taking a look at our changes.

File: vendor/src/github.com/docker/libnetwork/drivers/routed/netfilter.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+	for _, ipRange := range n.config.allowedRanges {
+		rules.addRule("-A", vethChainName, "-m", "iprange", "--src-range", ipRange.String(), "-j", "ACCEPT")
+	}
+
+	rules.addRule("-A", vethChainName, "-j", "CONTAINER-REJECT")
+
+	// Add JUMP in CONTAINERS, send all traffic going to the veth interface
+	rules.addRule("-I", containersChainName, "1", "-o", n.ifaceName, "-j", vethChainName)
+
+	if err := rules.apply(); err != nil {
```
Comment (jc-m) [inline vendor/src/github.com/docker/libnetwork/drivers/routed/netfilter.go:133]: Should the case where a rule with the same name already exist ? Could happen if a machine with containers reboot and get the same container.

Comment (jc-m) [inline vendor/src/github.com/docker/libnetwork/drivers/routed/netfilter.go:133]: Looks like the veth would change. ignore - good to go.

**Comment by erdody** [comment]
Thanks @mauri Added documentation in the docker README file.

**Pull request merged**